astropy/coordinates/sky_coordinate.py

```diff
@@ -1583,7 +1585,7 @@ def radial_velocity_correction(self, kind='barycentric', obstime=None,
 
     # Table interactions
     @classmethod
-    def guess_from_table(cls, table, **coord_kwargs):
+    def guess_from_table(cls, table, use_vo=True, **coord_kwargs):
```

Comment:
I am still not sure about having a `use_vo` option here. Is this a slippery slope? In the future, do we have to support `use_fits`, `use_asdf`, or whatever format-specific parsing here?